CONTRIBUTING.md

```diff
@@ -0,0 +1,373 @@
+# CONTRIBUTING
+
+Regardless if you are part of the core team or an external contributor, welcome and
+thank you for contributing to https://qiskit.org!
+
+In qiskit.org, we aim at creating an excellent work-space where all of us can feel
+welcomed, useful, respected and valued. If you are thinking to contribute to
+qiskit.org, you agree to abide by our [code of conduct](CODE_OF_CONDUCT.md) which
+we strongly recommend you read before continuing.
+
+Following these guidelines communicates you value the time and effort of the core
+contributors and maintainers of this site and so, thank you!
+
+## Start contributing
+
+This repository is for developing and maintaining https://qiskit.org, the website.
+If you want to contribute to Qiskit, the open-source Python library, you should go
+visit the [Qiskit repository](https://github.com/Qiskit/qiskit) instead.
+
+There are many ways of contributing: from catching a typo to coming up with a way
+of improving performance or accessibility; you can open an issue, or you can prepare
+a patch. In any case, read the contribution guidelines for opening new issues and
+submitting pull requests.
+
+Please, don't use this repository for asking questions about Qiskit: there are
+better ways to do it. Refer to our public [Slack](https://ibm.co/joinqiskitslack)
+or post a question in [stack overflow](https://stackoverflow.com/questions/tagged/qiskit).
+
+## Before you start
+
+Contributing to qiskit.org, and other Qiskit repositories, assumes you have some level
+of [Git](https://git-scm.com) knowledge. For external contributors, a basic understanding
+of repositories, remotes, branches and commits is needed. For core contributors, you
+should know about resolving conflicts and rebasing too.
+
+There are tons of useful resources about Git [out there](https://try.github.io/).
+
+## Opening issues
```

Comment:
This is great, thanks for going into this amount of detail because it's helpful from an onboarding perspective